python/shopping.py

```diff
@@ -0,0 +1,57 @@
+def main():
+    sum_of_buy = 0
+    shopping = {}
+    x = 0
+    price_so_far = 0
+    while(x == 0):
+
+        number = int(input(
```

Comment:
Will raise an exception if the input is not a number!
Add an input test that checks if the input is a number before converting it to an int.